kyakusahmed/EPIC-EMAIL
# Ft send message #165061873

What does this PR do?
User sends message

Description of Task to be completed?
Using java script fetch for a user to successfully send message to another user.

How should this be manually tested?
To test if the user has successfully sent a message, you should be able to see a "login successful" pop up
message.

Important Notes

link to [gh-pages](https://kyakusahmed.github.io/EPIC-EMAIL/templates/signup.html)
